Sources/Protocols/Reflectable.swift

```diff
@@ -38,7 +38,7 @@ extension NSObject: Reflectable {
             let properties = class_copyPropertyList(self, &count)
             var propertyNames: [Selector] = []
             for i in 0..<Int(count) {
-                if let propertyName = String(validatingUTF8: property_getName(properties?[i])) {
+                if let propertyName = String(validatingUTF8: property_getName((properties?[i])!)) {
```

Comment:
I'm typically not a fan of including implicitly unwrapped optionals. Could you pretty please remove it? You could do this as well:

```swift
if let currentProperty = properties?[i], let propertyName = String(validatingUTF8: property_getName(currentProperty))
```